src/main/java/frc/robot/subsystems/IntakePlacer.java

```diff
@@ -17,9 +19,10 @@
 
     public static final double MAX_SPEED = 0.5;
     public static final double MIN_SPEED = -0.1;
-    public static final double SERVO_START_ANGLE = 0;
-    public static final double SERVO_TARGET_ANGLE = 20;
+    public static final double SERVO_START_ANGLE = 110;
+    public static final double SERVO_TARGET_ANGLE = 0;
 
+    public final Supplier<Double> TARGET = rootNamespace.addConstantDouble("TARGET", 90);
```

Comment:
what's the difference between `SERVO_TARGET_ANGLE` and `TARGET`? Give better names.